ompi/mca/coll/base/coll_base_scatter.c

```diff
@@ -129,8 +129,8 @@ ompi_coll_base_scatter_intra_binomial(
         /* non-root, non-leaf nodes, allocate temp buffer for recv
          * the most we need is rcount*size/2 */
```

Comment:
Could you please update the comment to reflect the new code?

Reply:
@gpaulsen I believe the comment was right in the first place (but initially had the same idea, see my two previous comments :-))